mlir-tensorrt/CONTRIBUTING.md

```diff
@@ -0,0 +1,133 @@
+# Contributing to MLIR-TensorRT
+
+Contributions are welcome!
+
+## Coding Standard
+
+We will strive to follow the same coding standard as upstream MLIR, which is
+described here: https://llvm.org/docs/CodingStandards.html
+
+Python files are formatted using the [`black` formatter](https://black.readthedocs.io/en/stable/).
+
+## How to Submit a PR
```

Comment:
I am going to borrow this for Tripy contributing guide.

Reply:
Fork-based development requirements for Tripy are subject to CI workflow. CC: @pranavm-nvidia 
This should be a repository-wide requirement rather than project-specific to maintain consistency.

I think this requirement should eventually move to the [top-level contributing](https://github.com/NVIDIA/TensorRT-Incubator/pull/50) guideline since it affects both projects.